public/client.js

```diff
@@ -163,8 +160,9 @@ socket.on('playersReady', playersReady);
 socket.on('createPlayer', createPlayer);
 socket.on('noPlayer', playerNotExists);
 socket.on('updatePlayer', updatePlayer);
+socket.on('playGameAfterPlyerLeaves', playGame);
```

Comment:
Have you tested with 3 players? Or 4? If 1st player presses play, 2nd player leaves and 3rd and 4th don't do anything, will it start the game or will it wait for 3rd and 4th?

Reply:
it works for 2 players, but when I do it for 3 i get this error:
    if (Array.isArray(obj)) {
              ^

RangeError: Maximum call stack size exceeded

Reply:
not sure how to solve it, I google it and people say it's bc browser can not handle it